ipn/ipnlocal/c2n.go

```diff
@@ -348,6 +348,11 @@ func findCmdTailscale() (string, error) {
 			return ts, nil
 		}
 		return "", errors.New("tailscale.exe not found in expected place")
+	case "freebsd":
+		if self == "/usr/local/bin/tailscaled" {
+			return "/usr/local/bin/tailscale", nil
```

Comment:
Should we also stat it to make sure it's there?

Reply:
Yeah, the docs for `os.Executable` disclaim a hard guarantee, so it might be wise. I suppose the place to put such a check is after line 334.

Reply:
Refactored this function to always check that the returned tailscale path exists and is not a symlink. PTAL